clients/lsp-sqls.el

```diff
@@ -99,13 +101,26 @@
           :timeout lsp-sqls-timeout
           :range (list
                   :start (lsp--point-to-position
-                          (if (use-region-p)
-                              (region-beginning)
-                            (point-min)))
+                          (cond
+                           (start start)
+                           ((use-region-p) (region-beginning))
+                            (t (point-min))))
                   :end (lsp--point-to-position
-                        (if (use-region-p)
-                            (region-end)
-                          (point-max))))))))
+                        (cond
+                         (end end)
+                         ((use-region-p) (region-end))
+                         (t (point-max)))))))))
+
+(defun lsp-sql-execute-paragraph (&optional command)
+  "Execute COMMAND on paragraph against current database."
+  (interactive)
+  (let ((start (save-excursion
+		 (backward-paragraph)
+		 (point)))
+	(end (save-excursion
+	       (forward-paragraph)
+	       (point))))
+    (lsp-sql-execute-query command start end)))
```

Comment:
Can you remove tabs here and replace with spaces? And make sure the indentation is correct. I would prefer making `let` variables' lambda definition on the same line since this isn't very long. See below,

```el
  (let ((start (save-excursion (backward-paragraph) (point)))
	    (end (save-excursion (forward-paragraph) (point))))
```

Reply:
Thanks for reviewing. Done.